website/content/en/docs/tasks/operator.md

```diff
@@ -107,3 +106,48 @@ The `deploy peer` command has the following flags:
 - **tag:** This field determines the version of project images to pull. By default, it uses the `latest` version.
 
 All the flags are mapped to corresponding field in the ClusterLink custom resource.
+
+## Manual Deployment without CLI
+
+To deploy the ClusterLink project without using the CLI, follow the instructions below:
+
+1. Download the configuration files (CRDs, operator RBACs, and deployment) from GitHub:
+
+    ```sh
+    git clone git@github.com:clusterlink-net/clusterlink.git
+    ```
+
+1. Deploy ClusterLink CRDs:
+
+    ```sh
+    kubectl apply --recursive -f  ./clusterlink/config/crds
+    ```
+
+1. Deploy the ClusterLink operator:
+
+    ```sh
+    kubectl apply --recursive -f  ./clusterlink/config/operator
+    ```
+
+1. Convert the peer and fabric certificates to secrets:
+
+    ```sh
+    export CRT=<path to fabric certificates folder>
+    kubectl create secret generic cl-fabric -n clusterlink-system --from-file=ca=$CRT/cert.pem
```

Comment:
please use TLS secrets if possible.
If it also requires code changes, we should do the whole change in a different PR

Reply:
Should be part of https://github.com/clusterlink-net/clusterlink/issues/509